packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -14,19 +14,22 @@ import 'text_selection_toolbar_button.dart';
 import 'theme.dart';
 
 // Values extracted from https://developer.apple.com/design/resources/.
-// The height of the toolbar, including the arrow.
-const double _kToolbarHeight = 45.0;
+
 // Vertical distance between the tip of the arrow and the line of text the arrow
 // is pointing to. The value used here is eyeballed.
 const double _kToolbarContentDistance = 8.0;
+
+// The size of the arrow button to navigate between pages.
```

Comment:
I thought this is the arrow that `_clipPath` uses to create the arrow that points to the current selection?

Reply:
Ah, you're right there.